packages/jsii-pacmak/lib/targets/go/package.ts

```diff
@@ -336,9 +340,26 @@ export class InternalPackage extends Package {
       packageName,
       filePath,
       root.moduleName,
+      root.version,
       root,
     );
 
     this.parent = parent;
   }
 }
+
+function determineMajorVersionSuffix(version: string) {
+  const sv = semver.parse(version);
+  if (!sv) {
+    throw new Error(
+      `Unable to parse version "${version}" as a semantic version`,
+    );
+  }
+
+  // suffix is only needed for 2.0 and above
```

Comment:
Can we get a link to the go documentation about this?

Reply:
Done